public/pages/AnomalyCharts/containers/AnomalyHeatmapChart.tsx

```diff
@@ -109,28 +125,57 @@ export const AnomalyHeatmapChart = React.memo(
       });
 
       return [
-        COMBINED_OPTIONS,
+        getViewableCombinedOptions(
+          COMBINED_OPTIONS,
+          props.heatmapDisplayOption?.entityOption
+        ),
         {
           label: 'Individual entities',
-          options: individualEntityOptions,
+          options: individualEntityOptions.reverse(),
         },
       ];
     };
 
+    const getViewableCombinedOptions = (
+      existingOptions: any,
+      selectedCombinedOption: any | undefined
+    ) => {
+      if (!selectedCombinedOption) {
+        return existingOptions;
+      }
+      return {
+        label: existingOptions.label,
+        options: uniq([selectedCombinedOption, ...existingOptions.options]),
```

Comment:
Yes. The reason is I want to put the selected combined option as the 1st element of options, because we by default use 1st element as selected option as shown in [code](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/350/files#diff-a044985587be3c93885ad08b0bb90396e88600895b94ace004c30cda74f955d7R182)